### ISSUE TITLE
Type error in Typescript while adding a plugin

### ISSUE THREAD

**Issue by Elius94** [mistic100/Photo-Sphere-Viewer#807]
### Describe your problem

![image](https://user-images.githubusercontent.com/14907987/208678744-f32470d7-b5fe-4422-a7f3-75851f8b7812.png)

The type `AutorotatePlugin` is not assignable to `PluginConstructor` while I'm triyng to add a native AutorotatePlugin to the plugins array of the options.

It seems that There's a type mismatch in Typescript.
Thanks for your help

### Photo Sphere Viewer version

v5

**Comment by mistic100** [comment]
what about other plugins ?

**Comment by Elius94** [comment]
Sorry, I didn't was really shure that it's a bug!
I can't change it to bug report, can you?

To skip this error in a not nice way, I had to use this:

```ts
plugins: [[AutorotatePlugin as unknown as PluginConstructor, { autorotatePitch: "5deg" }], ...(options.plugins ? options.plugins : [])],
```

The other plugins array is an option provided from the top level React component. 
I'm working on the v5 migration for the React Wrapper that I develop.

However, it's the same also fi I use this alone.
I suppose that is a type definition mismatch.

Thanks

**Comment by mistic100** [comment]
My question was : does only this plugin has the issue or all of them ?

**Comment by mistic100** [comment]
This feature/bug fix has been released in [version 5.0.1](https://github.com/mistic100/Photo-Sphere-Viewer/releases/tag/5.0.1).
